scripts/GenerateFileCatalog.ps1

```diff
@@ -40,7 +40,9 @@ if(!$nuspecFiles) {
 }
 
 foreach($nuspecFile in $nuspecFiles) {
-    $moduleName = $nuspecFile.BaseName
+    # $moduleName = $nuspecFile.BaseName <- makes it be lowercase
```

Comment:
Is this sometimes necessary? It looks like you're using FullName now.

Reply:
I will keep it there as a reminder to not do it in this scenario